### ISSUE TITLE
Multiplayer Template: NetworkPlayerMovement Yaw and Pitch Quantized Out of Range

### ISSUE THREAD

**Issue by AMZN-Gene** [o3de/o3de-extras#73]
When NetworkPlayerMovement processes input to send across the network, the data is lost because the MouseAxis is in the range of [-1. 1] even though the user will move the mouse more than 1 or -1 pixels on screen during the game tick. This causes aiming to not be tied to how much I move the mouse, which is very confusing to players.

**Comment by drakeredwind01** [comment]
looks like this issue might be complete.
if only we had a way of showing that...
["triage-complete" label #15877](https://github.com/o3de/o3de/issues/15877)

**Comment by AMZN-Gene** [comment]
> looks like this issue might be complete. if only we had a way of showing that... ["triage-complete" label #15877](https://github.com/o3de/o3de/issues/15877)

Good call, it's been fixed in Multiplayer Sample. Now we need to update the Multiplayer Template to match Multiplayer Sample.
